### DISCUSSION TITLE
ytuner dnsmasq request per seconds

### DISCUSSION THREAD

**Discussion by TheBossME** [coffeegreg/YTuner#15] [Tips]
WARNING in dnsmasq core: Maximum number of concurrent DNS queries reached (max: 150)

The only dns server in ytuner.ini is pihole resolver service.

**Comment by coffeegreg** [comment]
Beatrice, Your observations and suggestions, as always, are valuable. :+1: Good to know.
In fact, YTuner has its own DNS resolver written and built using FreePascal, which has nothing to do with Pi-hole, but actually works on a similar principle.

**Comment by coffeegreg** [comment]
I think about unlock `Discussions` in YTuner repository. Perhaps this could be a useful channel for information exchange. :thinking:

**Comment by TheBossME** [comment]
Could you move this to discussions ?
